mis_builder/models/mis_report.py

```diff
@@ -147,18 +147,16 @@ class MisReportKpi(models.Model):
 
     _order = "sequence, id"
 
-    def name_get(self):
-        res = []
+    @api.depends("description", "name")
+    def _compute_display_name(self):
         for rec in self:
-            name = f"{rec.description} ({rec.name})"
-            res.append((rec.id, name))
-        return res
+            rec.display_name = f"{rec.description} ({rec.name})"
 
     @api.model
-    def name_search(self, name="", args=None, operator="ilike", limit=100):
-        domain = args or []
+    def _name_search(self, name, domain=None, operator="ilike", limit=None, order=None):
+        domain = domain or []
         domain += ["|", ("name", operator, name), ("description", operator, name)]
-        return self.search(domain, limit=limit).name_get()
+        return self.search(domain, limit=limit, order=order)
```

Comment:
This _name_search can be removed and replaced with `_rec_names_search = ["name", "description"]`.

Reply:
Done